test/ll/Vwhile05.ll

```diff
@@ -0,0 +1,15 @@
+declare void @_Bio__println (i64)
+define void @_B_main () {
+  %_0 = alloca i1, align 8
+  call void @_B_printWhile ()
+  store i1 0, i1* %_0, align 8
+  ret void
+}
+define internal void @_B_printWhile () {
+  %_0 = alloca i1, align 8
+  br label %L1
+  call void @_Bio__println (i64 0)
+  store i1 0, i1* %_0, align 8
+  ret void
+L1:
```

Comment:
@jclark this is the ll for while false statement. Is this intended as while false is redundant?

Reply:
Have you tested the output? I don't think that's valid `ll` syntax.

If you find bugs in the code generation, please create a separate issue.

Reply:
I fixed the bug for `while false`. You can remove this one.